echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/controllers/WebhooksController.groovy

```diff
@@ -61,11 +57,26 @@ class WebhooksController {
       }
     }
 
-    log.info("Webhook ${source}:${type}:${event.content}")
+    log.info("Webhook ${type}:${source}:${event.content}")
 
     if (sendEvent) {
       propagator.processEvent(event)
     }
   }
 
+  @RequestMapping(value = '/webhooks/{type}', method = RequestMethod.POST)
+  void forwardEvent(@PathVariable String type, @RequestBody Map postedEvent) {
+    Event event = new Event()
+    event.details = new Metadata()
+    event.details.type = type
+    event.content = postedEvent
+
+    if (event.content.source != null){
```

Comment:
Again space between ')' and '{'.

More importantly, what webhook provider sends a source in it's payload? I've only looked at Github thus far and didn't see it there.

If the source is null (and doesn't exist in event.content.source) ... is there any value in extracting something from the originating source http call, originating server or something perhaps?

@tomaslin is probably best to comment on whether this is impactful or not.